source/codegen/templates/service.h.mako

```diff
@@ -14,6 +14,7 @@ include_guard_name = service_helpers.get_include_guard_name(config, "_SERVICE_H"
 namespace_prefix = config["namespace_component"] + "_grpc::"
 if len(config["custom_types"]) > 0:
   custom_types = config["custom_types"]
+(input_custom_types, output_custom_types) = common_helpers.get_input_and_output_custom_types(functions)
```

Comment:
Instead of defining these variables all the way up here, you can do it on line 59 below. Same for `cpp.mako` file as well.

Reply:
If creating additional `<% %>` blocks hinders readability, this is fine too. I don't have strong opinion on this.